src/murmur/main.cpp

```diff
@@ -200,6 +205,114 @@ void cleanup(int signum) {
 	exit(signum);
 }
 
+auto parseCLI(int argc, char **argv) {
+	struct {
+		bool quit = false;
+		boost::optional< std::string > ini_file;
+		boost::tuple< std::string, boost::optional< int > > supw_srv;
+		boost::optional< int > disable_su_srv;
+		bool verbose_logging = false;
+#ifdef QT_NO_DEBUG
+		bool detach = true;
+#else
+		boost::logic::tribool detach = boost::logic::indeterminate;
+#endif
+		bool wipe_ssl   = false;
+		bool wipe_logs  = false;
+		bool log_groups = false;
+		bool log_acls   = false;
+
+		bool print_authors            = false;
+		bool print_license            = false;
+		bool print_3rd_party_licenses = false;
+
+#ifdef Q_OS_UNIX
+		bool limits = false;
+		boost::optional< int > read_supw_srv;
+#endif
+	} options;
+
+	CLI::App app;
+	app.set_version_flag("-v,--version", "Mumble server version " + Version::getRelease().toStdString());
```

Comment:
I think I'd prefer it if we only output the version itself. That'll make it easier for scripts to parse the version programmatically.